src/GafferOSL/ShadingEngine.cpp

```diff
@@ -95,25 +95,6 @@ struct TypeDescFromType<Color3f>
 	}
 };
 
-GeometricData::Interpretation geometricInterpretationFromVecSemantics( TypeDesc::VECSEMANTICS semantics )
```

Comment:
Yes, exactly. I think 3 of the different travis setups complained about this. The others were fine with it.